cfg/global_vars.mk

```diff
@@ -23,7 +23,7 @@ SNDFILES += ToxicRecvMessage.wav ToxicOutgoingCall.wav ToxicIncomingCall.wav
 SNDFILES += ToxicTransferComplete.wav ToxicTransferStart.wav
 
 # Install directories
-PREFIX = /usr/local
+PREFIX ?= /usr/local
```

Comment:
What's this change for? If the user sets a custom prefix, this will be over-written. Otherwise the default should be explicitly set by the makefile.